target/linux/layerscape/patches-6.12/900-layerscape-dts-ls1012a-frdm-add-gpio-export-for-PHY-.patch

```diff
@@ -0,0 +1,35 @@
+From cd61f354b9e5ed1897ccc8fcce731454e11c7543 Mon Sep 17 00:00:00 2001
+From: Pawel Dembicki <paweldembicki@gmail.com>
+Date: Fri, 31 Oct 2025 22:46:05 +0100
+Subject: [PATCH] layerscape: dts: ls1012a-frdm: add gpio-export for PHY RST
+ pin
+
+On LS1012A-FRDM board both PHY have connected RST pin to GPIO1_23.
+Let's make sure that it is up before PHY init.
+
+Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
+---
+ arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts | 12 ++++++++++++
+ 1 file changed, 12 insertions(+)
+
+--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
++++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
+@@ -31,6 +31,18 @@
+ 		clock-frequency = <24000000>;
+ 	};
+ 
++
++	gpio-export {
++		compatible = "gpio-export";
++
++		phy-rst {
++			/* both PHY have connected rst to GPIO1_23 */
++			gpio-export,name = "phy-rst";
++			gpio-export,output = <1>;
++			gpios = <&gpio0 23 1>;
++		};
++	};
```

Comment:
`gpio-export` is a OpenWrt specific feature. I would prefer `gpio-hog`. `gpio-hog` is supported upstream. The `gpio-export` is helpful when you want to change the GPIO from from user space, I understood that this is not needed here.

Reply:
Done.